src/ICSharpCode.SharpZipLib/Zip/FastZip.cs

```diff
@@ -558,6 +571,28 @@ private void ProcessFile(object sender, ScanEventArgs e)
 			}
 		}
 
+		// Set up the encryption method to use for the specific entry.
+		private void ConfigureEntryEncryption(ZipEntry entry)
+		{
+			if (!string.IsNullOrEmpty(Password))
+			{
+				switch (EntryEncryptionMethod)
```

Comment:
Codacy found an issue: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=5745489)